api/serverless.yml

```diff
@@ -86,6 +86,13 @@ functions:
                   path: ${self:custom.versions.v1}/publications/{id}
                   method: GET
                   cors: true
+    getPublicationSeed:
+        handler: src/components/publication/routes.getPubSeed
```

Comment:
Given the name, I can't really understand what is this route supposed to do ?!

Reply:
It gets the publication, for the seed script as it changes what information is required, compared to regular getPublication. I'm not sure what a more fitting name would be tbh 🤔

Reply:
Maybe something like "getSeedPublications" would be easier to understand